templates/terraform/resource.erb

```diff
@@ -387,7 +391,9 @@ if <%= props.map { |prop| "d.HasChange(\"#{prop.name.underscore}\")" }.join ' ||
         obj["<%= key[:fingerprint_name] %>"] = getRes["<%= key[:fingerprint_name] %>"]
 
 <%       end # unless key[:fingerprint_name] -%>
-<%      custom_update_properties_by_key(properties, key).each do |prop| -%>
+<%      custom_update_properties_by_key(properties, key)
+          .reject(&:url_param_only)
```

Comment:
I'm somewhat surprised this had to be added, and I wonder if it could be moved upstream somewhere. Probably doesn't have to happen in this PR, though.